rules/eslint/style/on.js

```diff
@@ -117,29 +117,29 @@ module.exports = {
     // disallow dangling underscores in identifiers
     "no-underscore-dangle": "off",
     // disallow the use of Boolean literals in conditional expressions
-    "no-unneeded-ternary": "off",
+    "no-unneeded-ternary": "error",
     // disallow whitespace before properties
-    "no-whitespace-before-property": "off",
+    "no-whitespace-before-property": "error",
     // enforce the location of single-line statements
-    "nonblock-statement-body-position": "off",
+    "nonblock-statement-body-position": ["error", "below"],
     // enforce consistent line breaks inside braces
-    "object-curly-newline": "off",
+    "object-curly-newline": ["error", { "minProperties": 2 }],
     // require or disallow padding inside curly braces
     "object-curly-spacing": ["error", "always"],
     // enforce placing object properties on separate lines
-    "object-property-newline": "off",
+    "object-property-newline": "error",
     // allow just one var statement per function
     "one-var": ["error", "never"],
     // require or disallow an newline around variable declarations
-    "one-var-declaration-per-line": "off",
+    "one-var-declaration-per-line": ["error", "initializations"],
     // require assignment operator shorthand where possible or prohibit it entirely
     "operator-assignment": ["error", "always"],
     // enforce operators to be placed before or after line breaks
-    "operator-linebreak": "off",
+    "operator-linebreak": ["error", "before"],
     // enforce padding within blocks
-    "padded-blocks": "off",
+    "padded-blocks": ["error", "never"],
     // require quotes around object literal property names
-    "quote-props": "off",
+    "quote-props": ["error", "as-needed", { "keywords": true, "numbers": true }],
```

Comment:
I am shocked this isn't in the "possible error" section. JavaScript keywords and numbers as object keys do NOT work well.